test/helpers.lua

```diff
@@ -674,6 +674,36 @@ local function write_file(name, text, no_dedent, append)
   file:close()
 end
 
+local function isCI()
+  local is_travis = nil ~= os.getenv('TRAVIS_CI_BUILD')
+  local is_appveyor = nil ~= os.getenv('APPVEYOR')
+  local is_quickbuild = nil ~= os.getenv('PR_NUMBER')
```

Comment:
@jszakmeister Is `$PR_NUMBER` env var defined in quickbuild? I couldn't find documentation for quickbuild env vars, and I don't see them in the quickbuild UI nor the build log.

Reply:
Correct.  I can manually add environment variables into the build and execute steps, but there's nothing available by default.